guide/additional-features/reloading-commands.md

```diff
@@ -0,0 +1,64 @@
+# Reloading Commands
+
+When writing your commands, you may find it tedious to restart your bot every time for testing the smallest changes. With a command handler, you can eliminate this issue and reload your commands while your bot is running.
+
+This section assumes you followed the [Command Handling](/guide/creating-your-bot/command-handling.md) part.
+
+::: warning
+ESM does not support require and clearing import cache. You can use [hot-esm](https://www.npmjs.com/package/hot-esm) to import files without cache.
+:::
+
+```js {1}
+const { SlashCommandBuilder } = require('discord.js');
+
+module.exports = {
+	data: new SlashCommandBuilder()
+		.setName('reload')
```

Comment:
i dont know if leaving it to use for everyone is a good idea
some warning etc should be here